universal-login-contracts/src/beta2/deployFactory.ts

```diff
@@ -1,6 +1,6 @@
 import {Wallet, ContractFactory} from 'ethers';
 import {TransactionOverrides} from '@universal-login/commons';
-import WalletProxyFactory from '../../build/WalletProxyFactory.json';
+import WalletProxyFactory from '../../contracts/WalletProxyFactory.json';
```

Comment:
set directory to `/dist/contracts` and 
copy-paste key Json files after build to `/src/beta2/contracts` to